src/components/Playground.tsx

```diff
@@ -68,7 +71,8 @@ class Playground extends React.Component<IPlaygroundProps, PlaygroundState> {
         hasShareButton: true,
         isRunning: this.props.isRunning,
         queryString: this.props.queryString,
-        sourceChapter: this.props.sourceChapter
+        sourceChapter: this.props.sourceChapter,
+        sourceLibrary: this.props.sourceLibrary
```

Comment:
I really think this should be renamed `externalLibrary`. If no objections, please remember to change the directory name under ./public/ where these libraries are stored, as well as the state property, etc.